com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -640,7 +640,8 @@ static void SetSupportedRenderingFeatures()
                 lightProbeProxyVolumes = false,
                 motionVectors = false,
                 receiveShadows = false,
-                reflectionProbes = true,
+                reflectionProbes = false,
```

Comment:
Note that we also need to change these values if we want to change the default values.

Reply:
`[SupportedRenderingFeatures.reflectionProbes](https://docs.unity3d.com/ScriptReference/Rendering.SupportedRenderingFeatures-reflectionProbes.html)` only decides whether the reflection probe option on Renderers should be displayed in the inspector. As we do not support all the modes here we want to opt out on having this displayed in URP. 
`reflectionProbesBlendDistance` on the other hand we need to enable the blend distance field in the ReflectionProbe inspector.